snippets/go.json

```diff
@@ -155,6 +155,18 @@
 		"test function": {
 			"prefix": "tf",
 			"body": "func Test$1(t *testing.T) {\n\t$0\n}"
+		},
+		"init function": {
+			"prefix": "finit",
```

Comment:
Given that 'test function' uses `tf` and 'goroutine function' uses `gf`, maybe the main function can use `mf` while the 'init' function can use `init` (`if` might not be the best choice :slightly_smiling_face:)